### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by willietran**
# Description

Fixes and improvements changelog post (June 27th). Post by Willie & Iain

# Links

N/A

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

File: about/changelog.md [hunk 1]
```diff
@@ -8,6 +8,16 @@ keywords: [changelog, upgrades, updates, releases]
 
 All the latest features and updates to Timescale products.
 
+## :bug: Fixes and improvements
```
Comment (billy-the-fish) [inline about/changelog.md]: ## ⚡ Performance improvements for live migration to Timescale Cloud

The following improvements have been made to the Timescale [live-migration docker image](https://hub.docker.com/r/timescale/live-migration/tags):

- With updated table filtering, live-migration retries automatically if you have an error or backoff. 
- Improvements to pbcopydb increase performance and removes unhelpful warning messages.
- The user notification log enables you to always select the most recent release for a migration run.

For improved stability and new features, update to the latest [timescale/live-migration](https://hub.docker.com/r/timescale/live-migration/tags) docker image. To learn more, see the [live migration docs](https://docs.timescale.com/migrate/latest/live-migration/).

Comment (alejandrodnm) [reply about/changelog.md]: I think I wouldn't mention `deadlock and intermittent crashes`. What about instead we mention that a bug that stops live-migration from moving forward and causing it to display a warning message saying that the replay rate is 0 B/s was fixed?

Comment (alejandrodnm) [reply about/changelog.md]: Or maybe it's too specific?

Comment (billy-the-fish) [reply about/changelog.md]: Updated ☝️

**Comment by willietran** [comment]
@billy-the-fish RE: Rahil's comment, maybe let's just throw these under the Ollama post too. Thoughts?

**Review by willietran — COMMENTED** [review]
Looks good to me!

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**